entity-framework/core/miscellaneous/connection-strings.md

```diff
@@ -61,7 +61,16 @@ public class BloggingContext : DbContext
 
 ## ASP.NET Core
 
-In ASP.NET Core the configuration system is very flexible, and the connection string could be stored in `appsettings.json`, an environment variable, the user secret store, or another configuration source. See the [Configuration section of the ASP.NET Core documentation](https://docs.asp.net/en/latest/fundamentals/configuration.html) for more details. The following example shows the connection string stored in `appsettings.json`.
+In ASP.NET Core the configuration system is very flexible, and the connection string could be stored in `appsettings.json`, an environment variable, the user secret store, or another configuration source. See the [Configuration section of the ASP.NET Core documentation](https://docs.asp.net/en/latest/fundamentals/configuration.html) for more details.
+
+For instance, you can use the [Secret Manager tool](https://docs.microsoft.com/aspnet/core/security/app-secrets#secret-manager) to store your database password and then, in scaffolding, use a connection string that simply consists of `Name=<database-alias>`.
```

Comment:
Follow our guidelines for links. Base domain does not need to be specified for docs.microsoft.com.

Reply:
@smitpatel I'm not seeing this spelled out in the guidelines - can you point me to what rules you are following? And an example would be helpful. Note this is not in the entity-framework docs.

Reply:
https://github.com/dotnet/EntityFramework.Docs/issues/1900

Reply:
@smitpatel Created PR #2351 for this.